packages/rest/src/rest.server.ts

```diff
@@ -513,6 +534,17 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
   }
 
+  /**
+   * Mount static assets to the REST server
+   * @param path The path to serve the asset
+   * @param root The root directory
+   * @param options Options for serve-static
+   */
+  asset(path: string, root: string, options?: ServeStaticOptions) {
+    this._setupAssetsRouter();
```

Comment:
Should we use the same static router for all static files because right now each folder would get it's own.

Reply:
No, we have the same router - `this._assetsRouter`. It's the container for multiple middleware, each per root dir.